rs/bitcoin/kyt/btc_kyt_canister.did

```diff
@@ -64,6 +67,12 @@ type BtcNetwork = variant {
     testnet
 };
 
+type KytMode = variant {
+    AcceptAll;
+    RejectAll;
```

Comment:
Mostly it was because the existing KYT cannister support this, and our ckbtc minter at the moment heavily relies on this feature in its tests. So supporting the same feature in the new KYT canister is an easy way to avoid breaking those tests.

Reply:
Thanks @ninegua for the explanation! I think this part

> ckbtc minter at the moment heavily relies on this feature in its tests.

was the one I was missing. Would it make sense to add it to the PR description?

Reply:
Sure, just added to the description.